api/src/main/java/datawave/microservice/querymetric/BaseQueryMetric.java

```diff
@@ -944,6 +944,8 @@ public void setError(Throwable t) {
             this.setErrorCode(qe.getErrorCode());
             this.setErrorMessage(qe.getMessage());
         } else {
+            // '500-1' signifies an 'Unknown server error'. This serves as a catch-all error code.
+            this.setErrorCode("500-1");
```

Comment:
I would like to see this value be made a static constant  for BaseQueryMetric. Additionally, I would like some javadoc added for this method that covers the possible scenarios of what the error code will be.

Reply:
Good idea!